.clang-format

```
@@ -13,6 +13,8 @@
 BasedOnStyle: LLVM
 AlignConsecutiveMacros: AcrossComments
 AllowShortEnumsOnASingleLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortFunctionsOnASingleLine: false
 AttributeMacros:
```

Nit: would be nice to keep this alphabetically ordered.

```suggestion
AllowShortBlocksOnASingleLine: false
AllowShortEnumsOnASingleLine: false
AllowShortFunctionsOnASingleLine: false
AttributeMacros:
```